docs/gallery/analysis_examples/ramp_analysis.py

```diff
@@ -24,13 +25,15 @@
 sweep_info = ct.get_ephys_sweeps(specimen_id)
 
 # Build the data set and find the ramp sweeps
-data_set = AibsDataSet(sweep_info=sweep_info, nwb_file=nwb_file)
-ramp_table = data_set.filtered_sweep_table(stimuli=data_set.ontology.ramp_names)
+data_set = create_data_set(sweep_info=sweep_info, nwb_file=nwb_file)
+ramp_table = data_set.filtered_sweep_table(
+    stimuli=data_set.ontology.ramp_names
+)
 ramp_sweep_set = data_set.sweep_set(ramp_table.sweep_number)
 
 # Build the extractors (we know stimulus starts at 0.27 s)
```

Comment:
This is probably not true for the ramp stimulus either? Seems likely copied from the long-square example.It works because spikes don't happen until long after this time and the analysis doesn't do anything with the stimulus waveform. But things like latency would not be accurately calculated.